open-company/open-company-web
# Company description

card: https://trello.com/c/tCfi2W9z

To test:
- open a company
- click on the company logo in the top left corner
- you should see the company profile page
- change some data (make sure you change the company description
- every time you exit a field (or you change the currency) there should be a PATCH call
- reload the page to make sure the data were saved
- click again on the company logo in the top left corner to go back to the company page
- merge
